packages/react-vanilla-components/src/components/CheckBoxGroup.tsx

```diff
@@ -84,6 +84,7 @@ const CheckBoxGroup = (props: PROPS) => {
                   value={enums![index]}
                   onChange={changeHandler}
                   readOnly={readOnly}
+                  checked={value?.includes(enums?.[index])}
```

Comment:
Please add same check for the checkbox component https://github.com/adobe/aem-forms-headless-components/blob/main/packages/react-vanilla-components/src/components/CheckBox.tsx#L50

Reply:
added